demo/src/main/java/com/bluelinelabs/conductor/demo/MainActivity.java

```diff
@@ -31,7 +36,49 @@ protected void onCreate(Bundle savedInstanceState) {
 
         router = Conductor.attachRouter(this, container, savedInstanceState);
         if (!router.hasRootController()) {
-            router.setRoot(RouterTransaction.with(new HomeController()));
+
+            HomeController homeController = new HomeController();
+            router.setRoot(RouterTransaction.with(homeController));
+            handleIntentDataUri(homeController);
```

Comment:
Won't deep linking not work if you've already launched the app if you use this strategy?